test/anno_verify_func_test1/main.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <string.h>
+#include "../../Annotations/Annotations.h"
+
+char* MustCall("free") foo() { // no warning
+    char* x = (char*)malloc(15);
+    char* y = (char*)malloc(15);
```

Comment:
I'd add a comment here to note that the analysis should warn about both `x` and `y` in this function (but not for annotation verification reasons): each is leaked in the case when the other is returned. Alternatively, add `free(y)` before `return x` and vice-versa.